modules/test/playwright/tests/portlet-configuration-web/rolePermissions.spec.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+// @ts-ignore
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import {portletConfigurationPermissionsPageTest} from '../../fixtures/portletConfigurationPermissionsPagesTest';
+
+export const test = mergeTests(
+	loginTest(),
+    portletConfigurationPermissionsPageTest
+);
+
+
+test ('LPD-25265 search results should stay when form submitted', async ({
+    page,
+    portletConfigurationPermissionsPage,
+}) => {
+    await portletConfigurationPermissionsPage.goto();
+    await expect(
+        portletConfigurationPermissionsPage.searchBar
+    ).toBeVisible();
+    await page.waitForTimeout(500);
+    
+    await portletConfigurationPermissionsPage.searchBar.click();
+    await portletConfigurationPermissionsPage.searchBar.fill('r');
+    await portletConfigurationPermissionsPage.searchBar.press('Enter');
+    await expect(
+        portletConfigurationPermissionsPage.resultsBanner
+    ).toBeVisible();
+    await expect(portletConfigurationPermissionsPage.ownerRoleCell).toBeVisible();
+    await expect(portletConfigurationPermissionsPage.siteMemberRoleCell).toBeVisible();
+
+    await portletConfigurationPermissionsPage.switchDeltaFrom20To4();
+    await expect(portletConfigurationPermissionsPage.ownerRoleCell).toBeVisible();
+    await expect(portletConfigurationPermissionsPage.siteMemberRoleCell).toHaveCount(0);
+    
+    await portletConfigurationPermissionsPage.saveButton.click();
+	await expect(portletConfigurationPermissionsPage.successMessage).toBeVisible();
+
```

Comment:
no empty line (all the expect in the same "block")

Reply:
I think this one is still missing

Reply:
Sorry about that, you're right, please see: dd1a79e96a3c9b547de7829c48653789d0906bca